scripts/install/wireguard.sh

```diff
@@ -44,36 +44,27 @@ function _install_wg () {
 		check_ubuntu_updates
 	fi
 
-	echo "Fetching APT updates"
 	apt_update
-	echo "Installing Wireguard from APT"
 	apt_install --recommends wireguard qrencode
 
-
 	if [[ ! -d /etc/wireguard ]]; then
 		mkdir /etc/wireguard
 	fi
 
 	chown -R root:root /etc/wireguard/
 	chmod -R 700 /etc/wireguard
-	# echo ""
-	modprobe wireguard >> $log 2>&1
-
-	if [[ $? != "0" ]]; then
-		echo "Could not modprobe Wireguard, script will now terminate."
-        echo "Please ensure a kernel headers package is installed that matches the currently running kernel."
-        echo "Currently running kernel:"
-        echo "$(uname -r)"
-        echo "Installed kernel headers:"
-        echo "$(dpkg -l | awk '{print $2}' | grep headers | grep amd64 | grep -v linux-headers-amd64 | sed 's/^/  '/g)"
-        echo "You may be able to resolve this error with \`apt install linux-headers-$(uname -r)\` or a system reboot."
-        echo "If you are using a custom kernel, your package names may differ."
-		echo "Please consult the swizzin log for further info if required."
+
+	if ! modprobe wireguard >> $log 2>&1 ; then
+		echo_error "Could not modprobe Wireguard, script will now terminate."
+        echo_info "Please ensure a kernel headers package is installed that matches the currently running kernel.
+Currently running kernel:\t\t$(uname -r)
+Installed kernel headers:\t\t$(dpkg -l | awk '{print $2}' | grep headers | grep amd64 | grep -v linux-headers-amd64 | sed 's/^/  '/g)
```

Comment:
Should there be a newline here? Testing this output gives me:

```
echo  -e "Installed kernel headers:\t\t$(dpkg -l | awk '{print $2}' | grep headers | grep amd64 | grep -v linux-headers-amd64 | sed 's/^/  '/g)"
```

```
Installed kernel headers:                 linux-headers-4.19.0-11-amd64
  linux-headers-4.19.0-9-amd64
```

Reply:
Don't think so, that's all on one line, no? think the command should change the whitespace between the versions

Reply:
The original command was structured like this:

```
echo "Installed kernel headers:"
echo "$(dpkg -l | awk '{print $2}' | grep headers | grep amd64 | grep -v linux-headers-amd64 | sed 's/^/  '/g)"
```

So that all the headers were printed under the section heading, the change you've made has put the first installed header on the same line.